lib/vimaly/client.rb

```diff
@@ -86,7 +86,11 @@ def bin(name_or_regex)
 
     def bins
       @bins ||= begin
-        get('/bins?max-results=500').map do |bin_data|
+        per_page = 500
+        data = (0..1).inject([]) do |list, page|
+          list += get("/bins?max-results=#{per_page}&page-token=#{page * 500}")
+        end
+        data.map do |bin_data|
```

Comment:
I guess I _might_ suggest this and discarding the `data` constant altogether.  In Javascript I would, but alas.  Ignore me!

```suggestion
        .map do |bin_data|
```